sockeye/arguments.py

```diff
@@ -173,20 +194,13 @@ def add_model_parameters(params):
                               help='Initialize model parameters from file. Overrides random initializations.')
 
     model_params.add_argument('--num-words',
-                              type=int_greater_or_equal(0),
-                              default=50000,
-                              help='Maximum vocabulary size. Default: %(default)s.')
-    model_params.add_argument('--num-words-source',
-                              type=int_greater_or_equal(0),
-                              default=None,
-                              help='Maximum source vocabulary size. Overrides --num-words. Default: %(default)s')
-    model_params.add_argument('--num-words-target',
-                              type=int_greater_or_equal(0),
-                              default=None,
-                              help='Maximum target vocabulary size. Overrides --num-words. Default: %(default)s')
+                              type=two_ints(greater_or_equal=0),
+                              default="50000:50000",
```

Comment:
this comment applies to all the defaults: why not leave the default values at containing a single value (in order to use the same for source and target)?